homeassistant/components/ecovacs.py

```diff
@@ -0,0 +1,87 @@
+"""Parent component for Ecovacs Deebot vacuums.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/ecovacs/
+"""
+
+import logging
+import random
+import string
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers import discovery
+from homeassistant.const import CONF_USERNAME, CONF_PASSWORD, \
+    EVENT_HOMEASSISTANT_STOP
+
+REQUIREMENTS = ['sucks==0.9.0']
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = "ecovacs"
+
+CONF_COUNTRY = "country"
+CONF_CONTINENT = "continent"
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.Schema({
+        vol.Required(CONF_USERNAME): cv.string,
+        vol.Required(CONF_PASSWORD): cv.string,
+        vol.Required(CONF_COUNTRY): vol.All(vol.Lower, cv.string),
```

Comment:
Use `vol.Lower()`. Probably have to disable pylint warning.

Reply:
Changing `vol.Lower` to `vol.Lower()` doesn't work due to the missing parameter. The way I have it here works and seems to be the way it's used elsewhere. Can you clarify what change you're proposing here?

Reply:
Does it actually not work or is it just pylint complaining. Last time I read the voluptuous docs it's `vol.Lower()` that is correct.

Reply:
It throws an exception; not just a linting error
```TypeError: Lower() missing 1 required positional argument: 'v'```